config/main.py

```diff
@@ -6582,6 +6591,18 @@ def subintf_vlan_check(config_db, parent_intf, vlan):
                     return True
     return False
 
+def is_subintf_shortname(intf):
```

Comment:
Can this be simple as below? Since we are already validating if intf starts with Po or Eth I believe we can just check if the name is full or else return false
```
if intf.startswitch("Ethernet") or intf.startswith("PortChannel"):
    return False
return True
```

Reply:
Done